### DISCUSSION TITLE
[Feature Request] Editable dropdown (Select variant?)

### DISCUSSION THREAD

**Discussion by SushritPasupuleti** [tamagui/tamagui#221] [Ideas]
A variant of the select component where you can enter text and have it filter out the options for you or just take you to the entry (currently this works on web out-of-the-box).

A reference would be [this Material-UI autocomplete component as they call it](https://mui.com/material-ui/react-autocomplete/)

Would be happy to assist with creation of this as well!

**Comment by leifdejong** [comment]
At least some component that can do a remote call to populate a selectable list. Perhaps this could be adding input functionality to `Select` like `Select.Input` in replacement of `Select.Value`.
